davinci_api-pingone_protect-reg-authn-pkg/terraform/vars.tf

```diff
@@ -22,6 +22,11 @@ variable "admin_env_id" {
   description = "P1 Environment containing the Worker App"
 }
 
+variable "PINGONE_ENVIRONMENT_ID" {
```

Comment:
this name doesn't appear to match description, but if that is what was intended just checking. Also the case is all upper while others are all lower.